### ISSUE TITLE
How does the border works?

### ISSUE THREAD

**Issue by tiagomsmagalhaes** [sharkdp/bat#439]
Is it possible to replicate with shell commands?

**Comment by sharkdp** [comment]
Yes. It's just unicode symbols from the "box drawing" block: https://unicode-table.com/en/blocks/box-drawing/
